queries.sql

```diff
@@ -5,4 +5,36 @@ SELECT date_of_birth from animals where name = 'Agumon' or name = 'Pikachu'
 SELECT name, escape_attempts from animals where weight_kg > 10.5
 SELECT * from animals where neutered is true
 SELECT * from animals where name != 'Gabumon'
-SELECT * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
\ No newline at end of file
+SELECT * from animals where weight_kg >= 10.4 and weight_kg <= 17.3
+
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified'
+ROLLBACK;
+
```

Comment:
- [ ] Kindly verify the change was undone using the following line at the end 👇🏽 

`SELECT species from animals;`